packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx

```diff
@@ -12,6 +12,17 @@ interface IProps {
   }[];
 }
 
+const shortNum = (n: number, fixed?: number) => {
```

Comment:
We have Utils.printCombatNumber that does something similar. Can we use that here?